helm/kvm-operator/templates/configmap.yaml

```diff
@@ -33,7 +33,7 @@ data:
         ntp:
           servers: {{ .Values.Installation.V1.Provider.KVM.NTP.Servers }}
         {{- if .Values.Installation.V1.Guest }}
-        tenant:
+        workload:
```

Comment:
This is the only change I could conceivably see having an impact. As far as I can tell, we don't override this in `installations` for any customer though.